client/.eslintrc.json

```diff
@@ -0,0 +1,29 @@
+{
+  "env": {
+    "es6": true,
+    "node": true,
+    "jest": true
+  },
+  "parserOptions": {
+    "ecmaVersion": 2017
```

Comment:
NIT: I don't know what I'm talking about, seems like from reading in https://eslint.org/docs/user-guide/configuring that 2018 does exist. 
Maybe a JS PR I could handle :P

Reply:
This was something set originally with the feathersjs generator, I don't know enough about how ECMAScript 2018 relates to functionality provided by Node 9. There are some things in newer versions of ECMAScript which are "Experimental" in Node 9, but that doesn't mean we should rely on them yet.